cli.py

```diff
@@ -129,15 +134,20 @@ def test(
     answers and produce replies.
     """
     logging.disable(logging.WARNING)
-    hparams_file = experiment + "hparams.yaml"
+    hparams_file = os.path.join(experiment, "hparams.yaml")
```

Comment:
So that a user can use as argument "path_to_folder" instead of "path_to_folder/"